src/external/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ function(check_submodules_present)
 endfunction()
 check_submodules_present()
 
+if(NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
```

Comment:
Put a comment above this line:
```cmake
# Fix a glslang hack
# See https://github.com/KhronosGroup/glslang/issues/1015 and https://github.com/Vita3K/Vita3K/pull/369 for details
```

Reply:
Done, i am just not sur for option set on OFF, but if that is not needed is good, because actually master using this install ^^